Editing Existing content v8 version (#2248)
umbraco/UmbracoDocs: Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Editing-Existing-Content/index.md

---
versionFrom: 8.0.0
---

# Editing Existing Content

## Content Within the Tree View

This documentation is intended as a guide that will be helping you gain an overview of how you edit in excisting content.

1.	When you are looking to edit content, the first thing you should do is locate the ***page*** you want to edit content on. This can be done in the section tree on the left side of the screen.

![Pick the page you want to edit in the section tree.](images/Pick-page.png)

2.	When you have located the page, you want to ***edit*** and clicked onto it.

> You will be presented with the content of the page (if there is any). 

Reviewer: ```suggestion

You will be presented with the content of the page (if there is any). 
```

> This content will be in the middle of the page (see image) the content is everything inside the red box, in this case there is three kinds of content.

Reviewer: ```suggestion
This content will be in the middle of the page (see image). The content is everything inside the red box. On this page there are three kinds of fields for content.
```

- A text box for the title.
- A media picker for the image.
- A rich text editor.

> These are just three of many kinds of content types.

Reviewer: ```suggestion
Which fields you see in this area and what type of content you will be able to fill in, is dependent on how the developers has built the site.
```

![This shows the content on the middle of the page.](images/Edit-content.png)

3.	After you have edited the content, you can click the save button to ***save*** the edits you did without publishing it.

![This shows the save button in the buttom left of the screen.](images/Save.png)

4.	At this point it is recommended that you ***preview*** the changes that you did, which you can do by pressing the preview button located to the left of the save button.

![This shows the preview button next to the save button.](images/Preview.png)

> 5.	Finally, after you have done the preview and confirmed that the changes are how you like it.

Reviewer: This sentence is a bit wierd. Wouldn't it make more sense, if this one if merged with the one on the next line? 🤔

```suggestion
5.	Finally, after you have done the preview and confirmed that the changes are how you like it.
```

> You can click ***save and publish***, to publish the changes you did.

Reviewer: Should this be part of the sentence before it? 🤔

```suggestion
You can click ***save and publish***, to publish the changes you did.
```

![This shows the save and publish button.](images/Save-publish.png)

## List View Pages

Some of the pages in your website may use a list view style which shows child pages in a table format rather than the standard tree view.

The first screenshot below shows three child items below the **The news** page in the tree view. If the website were to have tens or even hundreds more content pages, the tree view would become very large and unmanageable.

![Shows the standard tree view.](images/Non-listview.png)

The screenshot below shows what a page with a listview could look like.

> When a page uses listview, there will be no standard tree for that page, insted there will be a table list when you click into the page.

Reviewer: Is it supposed to say "table" or "list" here? 🤔

```suggestion
When a page uses listview, there will be no standard tree for that page, instead there will be a table list when you click into the page.
```

![Shows the regular listview.](images/List-view.png)

> If you want another type of list view, there are another option if you click the button shown in the screenshot below.

Reviewer: ```suggestion
If you want another type of list view, there is another option if you click the button shown in the screenshot below.
```

![Shows the options for listview.](images/Options-for-listview.png)

> Selectiong this option will present you with boxes rather then a tabel.

Reviewer: ```suggestion
Selectiong this option will present you with boxes rather then a table.
```

> This can be a good way to keep track of your pages if you need a quick overlook.

Reviewer: ```suggestion
This can be a good way to keep track of your pages if you need a quick overview.
```

> Selectiong this option will present you with boxes rather then a table.

Reviewer: ```suggestion
Selecting this option will present you with boxes rather then a table.
```

![Shows the other option for listview.](images/Other-version-of-listview.png)

> The table will show a maximum of ten child pages at any one time. Pagination controls allow you to navigate to additional pages should you have more than ten. In the top right of the table there is a search facility which enables you search for specific pages. This enables you to quickly and easily locate pages without having to use the pagination. You can click the column titles to sort the list items. For example clicking the name column title will sort the list items alphbetically.

Reviewer: ```suggestion
The table will show a maximum of ten child pages at any one time. Pagination controls allow you to navigate to additional pages should you have more than ten. In the top right of the table there is a search facility which enables you search for specific pages. This enables you to quickly locate pages without having to use the pagination. You can click the column titles to sort the list items. For example clicking the name column title will sort the list items alphbetically.
```
